src/Umbraco.Core/Persistence/Repositories/Implement/DomainRepository.cs

```diff
@@ -8,45 +8,39 @@
 using Umbraco.Core.Models;
 using Umbraco.Core.Models.Entities;
 using Umbraco.Core.Persistence.Dtos;
+using Umbraco.Core.Persistence.Factories;
 using Umbraco.Core.Persistence.Querying;
 using Umbraco.Core.Scoping;
 
 namespace Umbraco.Core.Persistence.Repositories.Implement
 {
-    // TODO: We need to get a readonly ISO code for the domain assigned
-
```

Comment:
This is already fixed by doing a left join on the language table in `GetBaseQuery()`.